### ISSUE TITLE
Create video guide for Carnivore2

### ISSUE THREAD

**Issue by Wierzbowsky** [RBSC/Carnivore2#65]
We need a video guide for Carnivore2 showing how to generally use the cartridge (including Dual-Slot feature), steps to upload the firmware, BIOSes, add/remove entries to/from FlashROM, create configuration entries, use various utilities and so on.  The video should mention the special Boot Block and utilities for Arabic and Korean MSXs as well as instructions on how to remove CP/M and network module from KYBT1 and KYBT2.

Basically, this should be a video-based user guide based on our main document:
https://sysadminmosaic.ru/msx/private/carnivore2/carnivore2

The video should be in Russian, but with English subtitles. @Wierzbowsky can help with translation. The base firmware/software version for creating the video is 2.50.

@DJs3000 Please provide the script here for review.

**Comment by Wierzbowsky** [comment]
One of our manufacturers/distributors from Brazil has created a few video guides on "how to":
https://www.youtube.com/watch?v=aoTocyc6vnQ
https://www.youtube.com/watch?v=I8sS5u_q2c8
https://www.youtube.com/watch?v=FQGXBtUHIwE
https://www.youtube.com/watch?v=OzyuSDD3A_4

Assembling:
https://www.youtube.com/watch?v=MmjMBxxXnzY

The original guide is unlikely to be created due to time issues.
